tests/testthat/helper-scope.R

```diff
@@ -38,7 +38,7 @@ renv_tests_scope <- function(packages = character(),
   renv_scope_libpaths(library, scope = scope)
 
   # return path to project directory
-  invisible(project)
+  invisible(renv_path_normalize(project))
```

Comment:
To avoid troubles with this:

```
> c(tempdir(), normalizePath(tempdir()))
[1] "/var/folders/9n/3nxsgkrj0rlfx196kzdttpww0000gn/T//RtmpbpCVk1"       
[2] "/private/var/folders/9n/3nxsgkrj0rlfx196kzdttpww0000gn/T/RtmpbpCVk1"
```